docs/getting-started/getting-started-http-03-versioning.md

```diff
@@ -0,0 +1,43 @@
+---
```

Comment:
Feel like versioning should be the last thing in the guide

Reply:
I restructured the entire thing, I want to encourage users to think about versioning up front rather than an afterthought, it will save them pain in the long run.